lib/src/main/java/com/dynatrace/file/util/PollBasedFilePoller.java

```diff
@@ -59,21 +74,86 @@ public boolean fileContentsUpdated() {
     return changedSinceLastInquiry.getAndSet(false);
   }
 
+  private synchronized void initialPoll() {
+    try {
+      prevChecksum = getFileChecksum();
+      prevModMillis =
+          Files.getLastModifiedTime(absoluteFilePath, LinkOption.NOFOLLOW_LINKS).toMillis();
```

Comment:
Would there be an issue with following symlinks? This is also not consistent with the behavior of `poll`.